scripts/build_torch_wheels.sh

```diff
@@ -46,9 +46,9 @@ function install_cudnn {
     ln -s "$CUDNN_TGZ_PATH" "$CUDNN_FILE"
   fi
   tar xvf "$CUDNN_FILE"
-  sudo cp cuda/include/cudnn.h /usr/local/cuda/include
+  sudo cp cuda/include/cudnn.h /usr/include
   sudo cp cuda/lib64/libcudnn* /usr/local/cuda/lib64
-  sudo chmod a+r /usr/local/cuda/include/cudnn.h /usr/local/cuda/lib64/libcudnn*
+  sudo chmod a+r /usr/include/cudnn.h /usr/local/cuda/lib64/libcudnn*
```

Comment:
This is not right.
If we want to place CUDNN with `/usr` root, libraries should be placed in same root.

Reply:
This is why:

```
% docker run -it nvidia/cuda:10.2-cudnn7-devel-ubuntu18.04
Unable to find image 'nvidia/cuda:10.2-cudnn7-devel-ubuntu18.04' locally
10.2-cudnn7-devel-ubuntu18.04: Pulling from nvidia/cuda
7ddbc47eeb70: Already exists
c1bbdc448b72: Already exists
8c3b70e39044: Already exists
45d437916d57: Already exists
d8f1569ddae6: Already exists
902fc5ce8229: Already exists
ae1bb79c5cfc: Already exists
fa6605c8fe7a: Already exists
0508a679d339: Already exists
b146e75a4883: Already exists
Digest: sha256:d3f5d6e8fe105dadb55aef7c80191d6281bbd18989a05124b108b7c8a522a5ad
Status: Downloaded newer image for nvidia/cuda:10.2-cudnn7-devel-ubuntu18.04
root@5353ba189e6f:/# find / -iname cudnn.h
/usr/include/cudnn.h
```

Reply:
Where are the cudnn* libraries located?

Reply:
It looks like they're located here:

```
root@7a9790b2dde7:/# ls -lah /usr/lib/x86_64-linux-gnu/libcudnn*
lrwxrwxrwx 1 root root   29 Dec  5 22:53 /usr/lib/x86_64-linux-gnu/libcudnn.so -> /etc/alternatives/libcudnn_so
lrwxrwxrwx 1 root root   17 Oct 27  2019 /usr/lib/x86_64-linux-gnu/libcudnn.so.7 -> libcudnn.so.7.6.5
-rw-r--r-- 1 root root 439M Oct 27  2019 /usr/lib/x86_64-linux-gnu/libcudnn.so.7.6.5
lrwxrwxrwx 1 root root   32 Dec  5 22:53 /usr/lib/x86_64-linux-gnu/libcudnn_static.a -> /etc/alternatives/libcudnn_stlib
-rw-r--r-- 1 root root 413M Oct 27  2019 /usr/lib/x86_64-linux-gnu/libcudnn_static_v7.a
```

Are these the ones you're looking for?